public/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>OSU CS Applied Plan Portal</title>
+
+        <!-- This is a 3rd-party stylesheet to make available the font families to be used for this page. -->
+        <link href="https://fonts.googleapis.com/css?family=Roboto|Roboto+Slab:100" rel="stylesheet">
+
+        <!-- This is a 3rd-party stylesheet to include Font Awesome icons: http://fontawesome.io/ -->
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" media="screen">
```

Comment:
For security, we should always use [Subresource Integrity](https://www.srihash.org) for each of our `<link>` and `<script>`. Pass the `href`'s value to the website and use the new line with the generated SRI.